src/debug/debugUtils.ts

```diff
@@ -2,30 +2,38 @@ import * as vscode from "vscode";
 
 import { Kubectl } from "../kubectl";
 
-export async function promptForDebugPort(defaultPort: string): Promise<string> {
+async function promptForPort(promptMessage: string, defaultPort: string): Promise<string> {
     const input = await vscode.window.showInputBox({
-        prompt: `Please specify debug port exposed for debugging (e.g. ${defaultPort})`,
-        placeHolder: defaultPort
+        prompt: promptMessage,
+        placeHolder: defaultPort,
+        value: defaultPort
     });
 
     return input && input.trim();
 }
 
+export async function promptForDebugPort(defaultPort: string): Promise<string> {
+    return await promptForPort("What debug port your app is enabled on?", defaultPort);

```

Comment:
'Enabled' reads oddly to me here.  I prefer your original "Please specify debug port exposed for debugging", though I guess it is a bit tautologous.  "Please specify the port for application debugging" perhaps?

Reply:
Previous app port prompt message is a bit similar as the debugging port message, and easy to lead mistake. That's why i refine the message. Refining the app port prompt message is enough. Have reverted this message to the original.